src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java

```diff
@@ -473,7 +473,8 @@ private static void setLogFileName(String forUserPersona,
         LOGGER.info(logMessage);
         ReportPortalLogger.logDebugMessage(logMessage);
         System.setProperty("webdriver." + browserType + ".logfile", logFile);
-        addBrowserLogFileNameFor(forUserPersona, Platform.web.name(), browserType, logFile);
+        Platform plaform = Runner.getPlatform();
```

Comment:
why are we changing this? This class is supposed to be only for web execution

Reply:
For electron platform it will give null pointer error for below method
void assignNewPersonaForUser(String userPersona, String newUserPersona) {
        replaceCapabilitiesFor(userPersona, newUserPersona);
        replaceDriverFor(userPersona, newUserPersona);
        replaceAppNameFor(userPersona, newUserPersona);
        replacePlatformFor(userPersona, newUserPersona);
        replaceLogFileNameFor(userPersona, newUserPersona);
    }

For replaceLogFileNameFor method we have below line which will always take web as platform if by default provided
String currentKey = getKeyNameForDeviceLogs(userPersona, currentPlatform.name());